pkg/template/resource_test.go

```diff
@@ -320,3 +321,13 @@ func Test_ResolveBinding_error(t *testing.T) {
 		})
 	}
 }
+
+func Test_ApplyUIDToResourceTemplate(t *testing.T) {
+	uid := Uid()
+	rt := json.RawMessage(`{"name": "something-$(uid)"}`)
+	expectedRt := bytes.Replace(rt, uidMatch, []byte(uid), -1)
+	actualRt := ApplyUIDToResourceTemplate(rt, uid)
+	if diff := cmp.Diff(string(expectedRt), string(actualRt)); diff != "" {
+		t.Error(diff)
```

Comment:
Please follow the logging format of the other tests explaining what the Diff means:
```
t.Errorf("ApplyUIDToResourceTemplate(): -want +got: %s", diff)
```
Without the `-want +got`, it can be confusing to understand what the diff string is showing.